src/network-handlers/ripple-handler.ts

```diff
@@ -150,12 +155,102 @@ export class RippleHandler {
     });
   }
 
+  async submitCreateTicketTransaction(xrplSignatures: XRPLSignatures[]): Promise<string[]> {
+    return await this.withConnectionMgmt(async () => {
+      try {
+        const signedTransactionBlobs = xrplSignatures.find(
+          sig => sig.signatureType === 'createTicket'
+        )!.signatures;
+        const multisignedTransaction = multiSignTransaction(signedTransactionBlobs);
+
+        const submitCreateTicketTransactionResponse =
+          await this.client.submitAndWait(multisignedTransaction);
+
+        checkRippleTransactionResult(submitCreateTicketTransactionResponse);
+
+        const meta = submitCreateTicketTransactionResponse.result.meta;
+
+        if (!meta) {
+          throw new RippleError('Transaction Metadata not found');
+        }
+
+        if (typeof meta === 'string') {
+          throw new RippleError(`Could not read Transaction Result of: ${meta}`);
+        }
```

Comment:
I don't think we need to compare and check all this stuff. Meta is always present, no? do you expect meta to not exist or to not be a string?

Reply:
```
/** Transaction metadata, which describes the results of the transaction.
   *  Can be undefined if a transaction has not been validated yet. */
  meta?: TransactionMetadata<T> | string
  ```
  
I need to verify that it is defined and is an object before treating it as one.